### ISSUE TITLE
'links' is not recognized as an internal or external command,

### ISSUE THREAD

**Issue by iqfareez** [kariy/imaluum-cli#10]
First of all, I like your work. Thanks!

When I run the command below:
```
$ imaluum timetable 1 2022/2023
```
I got the following error:
```
Fetching timetable for Sem 1 2022/2023..
'links' is not recognized as an internal or external command,
operable program or batch file.
node:internal/errors:841
  const err = new Error(message);
              ^

Error: Command failed: links -width 90 -dump C:\Users\Iqmal\AppData\Local\Temp/imaluum-cli/timetable_table.html
'links' is not recognized as an internal or external command,
operable program or batch file.

    at checkExecSyncError (node:child_process:828:11)
    at Object.execSync (node:child_process:899:15)
    at IMaluumSubPage.extractTimetableAndResultTableElement (C:\Users\Iqmal\AppData\Roaming\nvm\v16.15.0\node_modules\imaluum-cli\src\imaluum.js:123:40)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async C:\Users\Iqmal\AppData\Roaming\nvm\v16.15.0\node_modules\imaluum-cli\src\executor.js:208:30 {
  status: 1,
  signal: null,
  output: [
    null,
    '',
    "'links' is not recognized as an internal or external command,\r\n" +
      'operable program or batch file.\r\n'
  ],
  pid: 13052,
  stdout: '',
  stderr: "'links' is not recognized as an internal or external command,\r\n" +
    'operable program or batch file.\r\n'
}
```

**Comment by kariy** [comment]
found 2 problems here: the program i use to parse HTML table to string, `links`, and also how I handle file paths.

the former is actually an external program, kinda dumb tbh... will have to find a way to parse HTML table -> formatted string.  got the latter fixed.

thanks for giving me a reason to return back to this project. lmao

**Comment by iqfareez** [comment]
Ahh i see. haha thanks bro 👍

**Comment by kariy** [comment]
fixed in version 0.4.0
